docs/modules/ROOT/pages/references/architecture/sli_reporting.adoc

```diff
@@ -0,0 +1,216 @@
+          "value": 0.4
+        },
+        {
+          "timestamp": "2025-07-08T14:00:00Z",
+          "value": 0.2
+        },
+        {
+          "timestamp": "2025-07-08T15:00:00Z",
+          "value": 0.0
+        }
+      ]
+
+    }
+  }
+}
+----
+
+=== Downtime reporting
+
+Downtime windows can be added manually via the Downtime Window API.
+However, in some cases we can automate the reporting of downtime windows, such as when the 3rd party maintains a status page.
+Generally, such status pages provide an RSS feed with information about downtimes.
+
+RSS feeds are only loosely standardized and it's hard to generalize a field mapping for this specific use of RSS.
+For that reason, it would be significant effort to build our own RSS reading service.
+Instead, RSS ingestion can be handled by a simple bash script that's regularly run in a Kubernetes CronJob on VSHN's central cluster.
+
+We can use multiple bash scripts for different sources of RSS feeds.
+The bash script sends the parsed outage time window to the Downtime Window API.
+
+If more sophisticated processing of the 3rd party provided data is required, the simple bash script can be replaced with literally anything else.
+
+=== Access to Data
+
+The adjusted SLI/SLO data provided by the SLO reporting service can be integrated into Grafana dashboards via the Grafana `Infinity` data source plugin.
+
+The plugin queries the Data Querying API, and can apply arbitrary transformations to the result.
+
+For example, the following transform extracts the rate, cumulative sum (`sum_over_time` equivalent), error budget burndown/burnup, and cluster availability percentage for each SLO:
+[source,jsonata]
+----
+$sort($map($keys($.sli_data), function($key) {(
+    return {
+        $key : $map($lookup($$.sli_data, $key).error_rate_1h, function($v, $i, $a) {(
+            $cumulative := $sum($filter($a, function($sv, $j) {
+                    $j <= $i
+                }).value);
+            $objective := $lookup($$.sli_data, $key).objective;
+            $error_budget := $lookup($$.sli_data, $key).error_budget;
+            return {
+                "key": $key,
+                "timestamp": $v.timestamp,
+                "value": $v.value,
+                "cumulative": $cumulative,
+                "objective": $objective,
+                "error_budget": $error_budget,
+                "burnup": $cumulative / $error_budget,
+                "burndown": 1 - $cumulative / $error_budget,
+                "availability": $objective + (1-$objective) * (1 - $cumulative / $error_budget)
+            }
+        )})
+    }
+)}).*, function($l,$r){
+    $l.timestamp > $r.timestamp
+})
+----
+
+Since it's only possible to query one cluster at a time, it's not possible to create single panels that show data for all clusters on the same graph.
+However, it's possible to create a repeat panel that automatically generates one panel per cluster, which should be sufficient for an overview.
```

Comment:
I figured this is good enough. Wdyt?

Reply:
we're already doing a repeat panel for the current "Cluster availability" dashboard, I don't see a need to do something else.